bin/amqp_client

```diff
@@ -27,6 +27,7 @@ class AmqpClient
     end
 
     def start!
+      make_pid_dir
       worker_count.times do |i|
```

Comment:
Spotted a troubleshooting section in the Readme that indicated the script would error if the pid directory didn't exist. So added a fix that will create it for us.